x-pack/plugins/actions/server/builtin_action_types/resilient/translations.ts

```diff
@@ -9,3 +9,19 @@ import { i18n } from '@kbn/i18n';
 export const NAME = i18n.translate('xpack.actions.builtin.case.resilientTitle', {
   defaultMessage: 'IBM Resilient',
 });
+
+export const WHITE_LISTED_ERROR = (message: string) =>
```

Comment:
The same comment as for Jira: we are working on remove whitelist phrase from our plugins, so, please rename it to ALLOWED_HOSTS_ERROR and apiAllowedHostsError